gtests/net/mptcp/fastopen/client-TCP_FASTOPEN_CONNECT-blocking-sendmsg.pkt

```diff
@@ -0,0 +1,17 @@
+// Send data in SYN with a blocking sendto()
+--tolerance_usecs=100000
+`../common/defaults.sh
+ sysctl -q net.ipv4.tcp_fastopen=0x5`
+
+ 0.0      socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0      fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
++0.0      setsockopt(3, SOL_TCP, TCP_FASTOPEN_CONNECT, [1], 4) = 0
++0.0      connect(3, ..., ...) = 0
+
+// Switch to "blocking" mode, only the msk should be "blocked"
++0.0      fcntl(3, F_SETFL, O_RDWR) = 0
++0...0.4  sendto(3, ..., 500, 0, ..., ...) = 500
+
++0.0        >  S   0:500(500)                           <mss 1460, sackOK, TS val 100 ecr 0,   nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0        <  S.  0:0(0)          ack 501   win 65535  <mss 1460, sackOK, TS val 700 ecr 100, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey=2]>
```

Comment:
this should be:
```
+0.4      < S. #...
```

Reply:
This suggested change works with [mptcp: fix abba deadlock on fastopen v3](https://lore.kernel.org/mptcp/7c61250080c052b9dfb42a023c85dc36d4450daf.1665584136.git.pabeni@redhat.com/) for me

Reply:
Of course, thank you for the message! Fixed now!